sonar-wrapper.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```

Comment:
please move this file to ".ci" folder too and make it executable before pushing to repository to let git store this attrtibute and ease its usage .
Please make a comment here that file is for manual execution on local only.